hrm/department_app/models/employee_model.py

```diff
@@ -0,0 +1,39 @@
+"""Django models"""
+from django.db import models
+from department_app.models import department_models
+from djmoney.models.fields import MoneyField
+
+class Employee(models.Model):
+    """Employees"""
+    name = models.CharField('Name',max_length=50,primary_key=True)
+
+    department = models.ForeignKey(department_models.Department,on_delete=models.SET_NULL,null=True)
+
+    date = models.DateField('Date of birthday')
+
+    salary = MoneyField('Salary',max_digits=14, decimal_places=2, default_currency='USD')
+
+    email = models.EmailField('Email')
+
+    Active= "Active"
+
+    Inactive = "Inactive"
+
+    STATUS_CHOICES = (
+        (Inactive, "Inactive"),
+        (Inactive, "Inactive"),
+    )
+
+    status = models.CharField(max_length=10,
```

Comment:
Why 10? The maximum value of the field is Inactive and consists of 8 characters.